src/Lumi/Components/Form/Validation.purs

```diff
@@ -161,13 +161,13 @@ setModified = hmap (ModifyValidated (Modified <<< view _Validated))
 -- | records containing `Validated` values.
 newtype ModifyValidated = ModifyValidated (Validated ~> Validated)
 
-instance modifyValidated :: Mapping ModifyValidated (Validated a) (Validated a) where
-  mapping (ModifyValidated f) = f
+instance modifyValidated :: Mapping ModifyValidated a a => Mapping ModifyValidated (Validated a) (Validated a) where
+  mapping m@(ModifyValidated f) = over _Validated (mapping m) <<< f
 else instance modifyValidatedRecord :: (RL.RowToList r xs, MapRecordWithIndex xs (ConstMapping ModifyValidated) r r) => Mapping ModifyValidated {| r} {| r} where
   mapping d = hmap d
-else instance modifyValidatedArray :: (RL.RowToList r xs, MapRecordWithIndex xs (ConstMapping ModifyValidated) r r) => Mapping ModifyValidated (Array {| r}) (Array {| r}) where
-  mapping d = map (hmap d)
-else instance modifyValidatedA :: Mapping ModifyValidated a a where
+else instance modifyValidatedArray :: Mapping ModifyValidated a a => Mapping ModifyValidated (Array a) (Array a) where
+  mapping d = map (mapping d)
+else instance modifyValidatedIdentity :: Mapping ModifyValidated a a where
```

Comment:
This PR also fixes the behavior of `setFresh` and `setModified` to work recursively in the form data structure.